architecture/porting_guidelines.rst

```diff
@@ -411,7 +411,7 @@ product. There are a couple of reasons for that.
 
 Because of this we always urge companies and device manufacturers making the end
 product to follow the security guidelines from the chipmaker they are basing
-their products on.
+their products on. Please refer to :ref:`platform_documentation`
```

Comment:
I would rephrase to `Refer also to ...` since that section does not cover all supported platforms, and is likely to do so in the future.

Reply:
Ok